src/mongo/db/pipeline/expression_test.cpp

```diff
@@ -2179,6 +2178,10 @@ TEST(ExpressionFromAccumulators, StdDevSamp) {
          {{}, Value(BSONNULL)}});
 }
 
+TEST(ExpressionFromAccumulators, Pow) {
+    assertExpectedResults("$pow", {{{Value(-1LL), Value(-5LL)}, Value(-1LL)}});
```

Comment:
Please add at least one case for a negative even exponent. In fact, it should be easy to add some for everything up to 5, just to show the pattern:
```
assertExpectedResults("$pow", {{{Value(-1LL), Value(0LL)}, Value(1LL)}});
assertExpectedResults("$pow", {{{Value(-1LL), Value(-1LL)}, Value(-1LL)}});
assertExpectedResults("$pow", {{{Value(-1LL), Value(-2LL)}, Value(1LL)}});
assertExpectedResults("$pow", {{{Value(-1LL), Value(-3LL)}, Value(-1LL)}});
assertExpectedResults("$pow", {{{Value(-1LL), Value(-4LL)}, Value(1LL)}});
assertExpectedResults("$pow", {{{Value(-1LL), Value(-5LL)}, Value(-1LL)}});
```

It also would be good to include some cases with regular integers (as opposed to `long long`s):
`assertExpectedResults("$pow", {{{Value(-1), Value(-5)}, Value(-1)}});`
`assertExpectedResults("$pow", {{{Value(-1), Value(-4)}, Value(1)}});`

Maybe put those above the ones with the `LL`s, since they're slightly easier to read.